ajax/libs/sql.js/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "sql.js",
+  "filename": "js/sql.js",
+  "version": "0.3.2",
+  "description": "SQLite library with support for opening and writing databases, prepared statements, and more. This SQLite library is in pure javascript (compiled with emscripten).",
+  "keywords": [
+    "sql",
+    "sqlite",
+    "stand-alone",
+    "relational",
+    "database",
+    "RDBMS",
+    "data",
+    "query",
+    "statement",
+    "emscripten",
+    "asm",
+    "asm.js"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/kripken/sql.js.git"
+  },
+  "author": "Alon Zakai",
+  "npmName": "sql.js",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "js/*sql!(*raw)*"
```

Comment:
The raw files looks not useful here, @pvnr0082t could you please confirm with the author about the files should be hosted here?